sqlline

```diff
@@ -37,7 +37,7 @@ if [ ! -f target/fullclasspath.txt ]; then
 fi
 
 CP=
-for module in core cassandra druid elasticsearch2 elasticsearch5 file mongodb server spark splunk geode example/csv example/function; do
+for module in core cassandra druid elasticsearch2 elasticsearch5 file mongodb server spark splunk geode example/csv example/function kafka; do
```

Comment:
Julian's comment:
> you've put kafka at the end, but it should be in alphabetical order

Reply:
Putting it at the end is, of course, the modest thing to do.

I learned a while ago that being modest (especially when editing files with lots of unit test methods) creates what DBMS folks call a hot-spot. Everyone adds at the end, so we get merge conflicts. Make your change in the most logical place (which is alphabetical if there is no other organizing principle).